File: src/main/java/com/chrisrm/idea/actions/MTWeebAction.java
package com.chrisrm.idea.actions;

import com.chrisrm.idea.actions.themes.MTAbstractThemeAction;
import com.intellij.openapi.actionSystem.AnActionEvent;

public class MTWeebAction extends MTAbstractThemeAction {

  @Override
  public boolean isSelected(final AnActionEvent e) {
    return ClubMemberManager.getInstance().weebShitOn();
  }

  @Override
  public void setSelected(final AnActionEvent e, final boolean state) {
    ClubMemberManager instance = ClubMemberManager.getInstance();
    instance.toggleWeebShit();
  }
}

File: src/main/java/com/chrisrm/idea/actions/MTWeebAction.kt
package com.chrisrm.idea.actions

import com.chrisrm.idea.actions.themes.BaseThemeAction
import com.intellij.openapi.actionSystem.AnActionEvent
import io.acari.DDLC.actions.ClubMemberOrchestrator

class MTWeebAction : BaseThemeAction() {

    override fun isSelected(e: AnActionEvent): Boolean =
            ClubMemberOrchestrator.weebShitOn()

    override fun setSelected(e: AnActionEvent, state: Boolean) {
        super.setSelected(e, state)
    }
}
